hashin.py

```diff
@@ -436,7 +436,10 @@ def get_latest_version(data, include_prereleases):
     all_versions = []
     count_prereleases = 0
     for version in data["releases"]:
-        v = parse(version)
+        try:
+            v = parse(version)
+        except InvalidVersion:
```

Comment:
This needs a code-comment to explain why it's doing this. 
I think you can take the title of this PR. 

By the way, should it print a warning when it happens?

Reply:
> This needs a code-comment to explain why it's doing this. I think you can take the title of this PR.

@peterbe I can sure add an in-code comment: done.

> By the way, should it print a warning when it happens?

I'm producing some output now. How do you feel about the new approach?